tests/plugins/autogrow/autogrowwidth.js

```diff
@@ -0,0 +1,37 @@
+/* bender-tags: editor, autogrow */
+/* bender-ckeditor-plugins: autogrow */
+
+( function() {
+	'use strict';
+
+	bender.editor = {};
+
+	bender.test( {
+		init: function() {
+			// Remove root editor element border.
+			var editorRootElement = CKEDITOR.document.getById( 'cke_test_editor' );
+			editorRootElement.setStyle( 'border', 'none' );
+		},
+
+		// #4286
+		'test editor width': function() {
+			var editor = this.editor,
+				bot = this.editorBot;
+
+			var html = '';
+			for ( var i = 0; i < 6; i++ ) {
+				html += '<p>test ' + i + '</p>';
+			}
+
```

Comment:
Maybe we can get editor width here and check if it's the same after editor autogrows (as a second check after `> 0`)?

Reply:
You could also check if height increased as a last check.